tests/test_categorical_encoding.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import pandas as pd
+from src.koheesio.pandas.categorical_encoding  import PandasCategoricalEncoding
+import unittest
+import pandas as pd
```

Comment:
you're importing pandas twice. Also, import pandas through the koheesio module (as stated above) to avoid conflict:
`from koheesio.pandas import pandas as pd`